### PR TITLE
Compass Cockpit - sub-chart (preview)

### PR DESCRIPTION
**Pull request by kwiatekus**
**Description**

Changes proposed in this pull request:

- Add preview Compass UI to console navigation

**Related issue(s)**
https://github.com/kyma-incubator/compass/issues/152

### CODE REVIEW

File: installation/scripts/minikube.sh [hunk 1]
```diff
@@ -219,7 +219,7 @@ function start() {
...
+    addDevDomainsToEtcHosts "apiserver.${MINIKUBE_DOMAIN} console.${MINIKUBE_DOMAIN} catalog.${MINIKUBE_DOMAIN} instances.${MINIKUBE_DOMAIN} brokers.${MINIKUBE_DOMAIN} dex.${MINIKUBE_DOMAIN} docs.${MINIKUBE_DOMAIN} add-ons.${MINIKUBE_DOMAIN} lambdas-ui.${MINIKUBE_DOMAIN} console-backend.${MINIKUBE_DOMAIN} minio.${MINIKUBE_DOMAIN} jaeger.${MINIKUBE_DOMAIN} grafana.${MINIKUBE_DOMAIN} log-ui.${MINIKUBE_DOMAIN} loki.${MINIKUBE_DOMAIN} configurations-generator.${MINIKUBE_DOMAIN} gateway.${MINIKUBE_DOMAIN} connector-service.${MINIKUBE_DOMAIN} oauth2.${MINIKUBE_DOMAIN} oauth2-admin.${MINIKUBE_DOMAIN} kiali.${MINIKUBE_DOMAIN} compass-gateway.${MINIKUBE_DOMAIN} compass.${MINIKUBE_DOMAIN} compass-mf.${MINIKUBE_DOMAIN}"
```
Comment (pkosiec) [inline installation/scripts/minikube.sh]: Please also remember about making PR to `cli` repo

File: resources/compass/values.yaml [hunk 2]
```diff
@@ -1,4 +1,7 @@
 global:
+  defaultTenant : 3e64ebae-38b5-46a0-b1ed-9ccee153a0ae
+  tenants: foo bar
```
Comment (pkosiec) [inline resources/compass/values.yaml]: What does the `foo bar` mean? Tenants should be in UUID format

File: installation/scripts/minikube.sh (hunk 1 above)
Comment (kwiatekus) [reply installation/scripts/minikube.sh]: done : https://github.com/kyma-project/cli/pull/192

File: resources/compass/values.yaml (hunk 2 above)
Comment (kwiatekus) [reply resources/compass/values.yaml]: Those are sample tenants for demonstrating multi-tenancy model. Those will be converted to UUID on the flight.

File: resources/core/charts/console/values.yaml [hunk 3]
```diff
@@ -37,7 +37,7 @@ cluster:
   # Provide a list of navigation nodes that should be hidden. For example 'namespace.operation.secrets settings foo.bar'
   disabledNavigationNodes: ''
   # Provide a list of Namespaces that should be considered as system Namespaces. The Console UI gives you the possibility to toggle the visibility of system Namespaces.
-  systemNamespaces: 'compass-system istio-system knative-eventing knative-serving kube-public kube-system kyma-installer kyma-integration kyma-system natss'
+  systemNamespaces: 'istio-system knative-eventing knative-serving knative-build kube-public kube-system kyma-backup kyma-installer kyma-integration kyma-system natss compass-system'
```
Comment (dariadomagala) [inline resources/core/charts/console/values.yaml]: Please revert this change
